### PR TITLE
OpenAPI 3.0 specs for user creation

### PR DESCRIPTION
**Pull request by andrewjouffray**
Specs written by Gustavo and edited by Andrew. Covers the user creation endpoint. 

See specs at http://localhost:9001/docs

Note: The endpoint is not yet implemented.

### CODE REVIEW

**Review by josarlo84 — APPROVED** [review]
Great job. You are right, we overlooked username is our meeting. Good catch.

**Comment by andrewjouffray** [comment]
We could also just use the email as login if needed.

File: static/oas/v1/users/users.yml [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
-    
\ No newline at end of file
+    post:
+      tags:
+        - user
+      summary: Creates a user
+      requestBody:
+        description: Create user object
+        content:
+          application/json:   # we will just use json for now
```
Comment (jak103) [inline static/oas/v1/users/users.yml:10]: We will only ever do json :)

File: static/oas/v1/users/users.yml [hunk 2]
```diff
@@ -1,3 +1,76 @@
...
+      password:
+        type: string
+        example: '123!@dsQ'
+      email:
+        type: string
+        example: john@email.com
+      phoneNumber:
+        type: string
+        example: '123-456-7890'
+      baseExperience: # will this match skill level?
```
Comment (jak103) [inline static/oas/v1/users/users.yml:43]: Remind me and we can talk about this in class.

**Pull request merged**